### PR TITLE
User Photo Feed: Completed HTML markdown

### PR DESCRIPTION
**Pull request by mserykh**
# Related Issue
[User Photo Feed: Create an HTML semantic layout](https://github.com/mserykh/instaclone/issues/8)

# Proposed Changes
- Created a directory with essential images and icons
- Created a markdown file for the `User Photo Feed` page

### CODE REVIEW

File: src/user-photo-feed.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                <li><a href="#">Jobs</a></li>
+                <li><a href="#">Privacy</a></li>
+                <li><a href="#">Terms</a></li>
+                <li><a href="#">Locations</a></li>
+                <li><a href="#">Top Accounts</a></li>
+                <li><a href="#">Hashtags</a></li>
+                <li><a href="#">Language</a></li>
+            </ul>
+        </nav>
+        <span>© 2020 Instagram from Facebook</span>
```
Comment (dev-experience) [inline src/user-photo-feed.html:118]: Please remove the copyright

File: src/user-photo-feed.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            <ul>
+                <li><a href="#">Posts</a></li>
+                <li><a href="#">IGTV</a></li>
+                <li><a href="#">Saved</a></li>
+                <li><a href="#">Tagged</a></li>
+            </ul>
+        </nav>
+
+        <article class="user-photo-feed">
+            <div><a href="#"><img src="./assets/user-photo-feed/img/user-picture-1.jpg" height="293" width="293" alt=""></a></div>
```
Comment (dev-experience) [inline src/user-photo-feed.html:86]: Isn't it a list?

File: src/user-photo-feed.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                <li class="main-navigation-item"><a href="#">Activity</a></li>
+                <li class="main-navigation-item"><a href="#">Profile</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <h1></h1>
+
+        <section>
```
Comment (dev-experience) [inline src/user-photo-feed.html:31]: I believe, `section` should be nested into an article or something.

File: src/user-photo-feed.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            <header>
+                <img src="./assets/user-photo-feed/img/profile_picture.jpg" height="150" width="150" alt="">
```
Comment (dev-experience) [inline src/user-photo-feed.html:33]: Empty `alt`?

File: src/user-photo-feed.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                            In love with Paris, Jazz, House, Bread💓<br>
+                            .<br>
+                            Alterego <a href="#">@trifle_on_a_stick</a></p>
+                    </div>
+                </section>
+            </header>
+        </section>
+        <div class="highlights">
+            <ul class="highlights-list">
+                <li class="highlights-item">
```
Comment (dev-experience) [inline src/user-photo-feed.html:56]: BEM?

File: src/user-photo-feed.html (hunk 2 above)
Comment (dev-experience) [inline src/user-photo-feed.html:86]: Why there are CSS classes in the highlights but not here?

Comment (mserykh) [reply src/user-photo-feed.html:86]: To be honest I am not sure.

File: src/user-photo-feed.html (hunk 3 above)
Comment (mserykh) [reply src/user-photo-feed.html:31]: @dev-experience  Not always. It could go separately.

File: src/user-photo-feed.html (hunk 1 above)
Comment (mserykh) [reply src/user-photo-feed.html:118]: @dev-experience OK

File: src/user-photo-feed.html (hunk 4 above)
Comment (mserykh) [reply src/user-photo-feed.html:33]: @dev-experience Yes, for now they are empty everywhere on the page. Should I add text?

File: src/user-photo-feed.html (hunk 5 above)
Comment (mserykh) [reply src/user-photo-feed.html:56]: @dev-experience Not yet. I was thinking to apply it in CSS.

File: src/user-photo-feed.html (hunk 2 above)
Comment (mserykh) [reply src/user-photo-feed.html:86]: @dev-experience I added some classes for now. I was thinking to work closer with styles while I will work on CSS.

Comment (mserykh) [reply src/user-photo-feed.html:86]: @dev-experience Is it better to put them into a list?

Comment (dev-experience) [reply src/user-photo-feed.html:86]: > @dev-experience Is it better to put them into a list?

Semantically it's a list.

File: src/user-photo-feed.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                <li class="main-navigation-item"><a href="#">Direct</a></li>
+                <li class="main-navigation-item"><a href="#">Explore</a></li>
+                <li class="main-navigation-item"><a href="#">Activity</a></li>
+                <li class="main-navigation-item"><a href="#">Profile</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <h1></h1>
```
Comment (dev-experience) [inline src/user-photo-feed.html:29]: ??

File: src/user-photo-feed.html (hunk 3 above)
Comment (dev-experience) [reply src/user-photo-feed.html:31]: Doesn't look good for me. But if you insist...

File: src/user-photo-feed.html (hunk 4 above)
Comment (dev-experience) [reply src/user-photo-feed.html:33]: `alt` is meant to have a text, yes.

File: src/user-photo-feed.html (hunk 2 above)
Comment (dev-experience) [reply src/user-photo-feed.html:86]: Doesn't look very consistent.

Comment (mserykh) [reply src/user-photo-feed.html:86]: @dev-experience OK, I will make it a list

File: src/user-photo-feed.html (hunk 3 above)
Comment (mserykh) [reply src/user-photo-feed.html:31]: @dev-experience I will leave it as is

File: src/user-photo-feed.html (hunk 4 above)
Comment (mserykh) [reply src/user-photo-feed.html:33]: @dev-experience I will add texts

File: src/user-photo-feed.html (hunk 2 above)
Comment (mserykh) [reply src/user-photo-feed.html:86]: @dev-experience Agree. OK, I will add classes.

File: src/user-photo-feed.html (hunk 6 above)
Comment (mserykh) [reply src/user-photo-feed.html:29]: @dev-experience I forgot about `h1`. There is no `h1` on the page so there must be one. I will create it and add to it "invisible" class.

**Comment by mserykh** [comment]
I made requested changes:
* Added texts to all `alt` atributes
* Added classes to all elements
* Deleted copyright from the footer
* Added text and class to `h1`
* I wrapped photo feed in a list `ul`
I also adjusted layout with the following:
* Added elements to every photo in the feed
* Added icons to the menu
* Added a wrapper to a main section

**Pull request merged**